inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -78,6 +80,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     eliminations->add_matcher<ngraph::pass::NopElimination>(); // may introduce fake dynamism
     eliminations->set_name("ngraph::pass::CommonEliminations");
 
+    manager.register_pass<ngraph::pass::BinarizeWeights>();
```

Comment:
Why you decided to call it before ConstantFolding. Does it depends on it?

Reply:
first version of this pass heavily depended on CF, but that's not the case anymore so I can probably move that closer to ConvToBinaryConv transformation.